TradeSkillMaster_Crafting/Modules/CraftingGUI.lua

```diff
@@ -252,33 +252,38 @@ function GUI:EventHandler(event, ...)
 			-- local unit, _, _, _, spellID = ... -- parameter ... doesn't provide spellID in 3.3.5a
 			local unit, spellName = ...
 			local spellID = TSM.SpellName2ID[spellName]
```

Comment:
`TSM.SpellName2ID[spellName]`
change
Might be used incorrectly in more locations.
```lua
TSM.SpellName2ID(spellName)
```